file.go

```diff
@@ -0,0 +1,36 @@
+package crypt
+
+import (
+	"fmt"
+	"io/ioutil"
+	"os"
+)
+
+// EncryptFile encrypts srcFile using cipher enc into dstFile.
+// DstFile must not exists.
+func EncryptFile(srcFile, dstFile string, key []byte, enc Cipher) error {
```

Comment:
If we are going to maintain this on the long run it would be cool to have a "Encrypt" that takes io.Reader/io.Writer =D

Reply:
Indeed